floodviz/static/js/map.js

```diff
@@ -32,6 +32,9 @@
 
 			var self = {};
 
+			//governs whether map=hydrograph interactions will be turned on
+			var embed = FV.embed
```

Comment:
So I believe the effect of FV.embed is to disable the interactions. Could we pass this into the map module in options and call it disableInteractions? This will make it clearer and then can also be potentially used by svg to png generation code.